python_modules/dagster/dagster/_core/definitions/partition_mapping.py

```diff
@@ -327,7 +327,14 @@ def get_downstream_partitions_for_partitions(
         return downstream_partitions_def.empty_subset().with_partition_keys(set(matching_keys))
 
 
-class StaticPartitionMapping(PartitionMapping):
+@whitelist_for_serdes
+class StaticPartitionMapping(
+    PartitionMapping,
+    NamedTuple(
+        "_StaticPartitionMapping",
+        [("downstream_partition_keys_by_upstream_partition_key", PublicAttr[Mapping])],
```

Comment:
Mind including the full parameterized type here?

Reply:
fixed